kafka_utils/kafka_cluster_manager/cmds/replace.py

```diff
@@ -50,8 +52,8 @@ def build_subparser(self, subparsers):
         subparser.add_argument(
             '--dest-broker',
             type=int,
-            required=True,
-            help='Broker id of destination broker.',
+            default=None,
+            help='Broker id of destination broker. If empty, will shrink the replica set of topics',
```

Comment:
nit: Although unlikely but what will happen if this script is run multiple times without a dest-broker such that the replicas for a topic become 0? Should we have a check that prevents that from happening?

Reply:
That is already checked, it'll fail in that case.

For example, if I do this on our test cluster (which only has 3 brokers, and some topics with only 1 replica), removing a broker will result in the following error:

```
ERROR:kafka_utils.util.validation:Non-empty "replicas" expected: {'topic': 'my_topic', 'partition': 4, 'replicas': []}
```

Reply:
Sounds good!